include/swift/ASTSectionImporter/ASTSectionImporter.h

```diff
@@ -35,6 +35,14 @@ namespace swift {
   /// \return true if successful.
   bool parseASTSection(MemoryBufferSerializedModuleLoader &Loader,
                        StringRef Data, const llvm::Triple &filter,
+                       llvm::raw_ostream &error,
```

Comment:
Should we use `llvm::Expected` instead?

Reply:
This seems fine to me personally as it matches the existing style on the lldb side (eg., DeserializeAllCompilerFlags in SwiftASTContext.cpp) But if it's preferred, I'd be happy to make that change.

Reply:
Done.